ui/app/adapters/keymgmt/provider.js

```diff
@@ -0,0 +1,35 @@
+import ApplicationAdapter from '../application';
+import { all } from 'rsvp';
+
+export default class KeymgmtKeyAdapter extends ApplicationAdapter {
+  namespace = 'v1';
+  listPayload = { data: { list: true } };
+
+  pathForType() {
+    return 'keymgmt/kms';
```

Comment:
`keymgmt` is actually a dynamic variable, which is the :backend param on the path

Reply:
Ahhhh ok I didn't catch that and overlooked the fact that `keymgmt` is autopopulated when you enable the secrets backend but you can actually name it whatever. Thanks!

Reply:
I refactored the adapter and decided to override the `buildURL` method to pull the backend value from either the snapshot or query in [this commit](https://github.com/hashicorp/vault/pull/13797/commits/014fefa3e6d41f2a1f3ed8840f196c3e0b2860fa).